tests/fixtures/encoder_decoder/nlvr_semantic_parser/grouped_experiment.json

```diff
@@ -6,8 +6,8 @@
   "vocabulary": {
```

Comment:
For ease of use with our predictor scripts (see `scripts/train_fixtures.py` on the master branch), it's actually helpful to have this stay as `some/path/experiment.json`.

Reply:
Good to know, done.